integrations/tensorflow/test/python/iree_tf_tests/layers/layers_test.py

```diff
@@ -298,7 +298,7 @@ def has_mutually_exclusive_kwargs(
     "Lambda":
         tf_test_utils.unit_test_specs_from_signatures(
             signature_shapes=UNARY_SIGNATURE_SHAPES,
-            kwargs_to_values=dict(function=[lambda x: x**2])),
+            kwargs_to_values=dict(function=[lambda x: x*x])),
```

Comment:
AFAICT the exact computation here doesn't matter at all (it's just testing lambda itself), so this seems fine